Travel_fellowships.md

```diff
@@ -1,41 +1,178 @@
-the following criteria. Make sure that you include sufficient information on the application form for us to
-judge each criterion.
+These fellowships are available to support both in-person and remote (virtual) participation at events such as conferences, workshops, training courses or collaborative development sprints.
 
-**Does the event qualify?** Does the event promote open source bioinformatics software
-development and open science in the biological research community? Events can include - but are not limited to - conferences, workshops, codefests, and hackathons. We give
-slight preference to OBF events and member projects.
+## Application deadlines
 
-The event can occur anytime in a one-year window starting one month after the application deadline.
+There are two rounds of applications with the following **application deadlines**:
 
-**What is your participation?** We look at both your role at the event, your past participation in the project that you are representing, and also your general history in open source development or open science. We give slight preference to
-participation in OBF events and member projects.
+- Round-1: 1 April
+- Round-2: 1 October
 
-**Increasing diversity**: Does your attendance increase diversity at the event? We understand increasing diversity as increasing participation from groups otherwise underrepresented in the community or at the event, including but not limited to underrepresented demographic groups, ethnic origins, career stages, and people with disabilities. We do not consider diversity of operating systems or text editors.
+The calls for applications are announced through a [blog post on the website](https://www.open-bio.org/blog/), [official mailing list](https://mailman.open-bio.org/mailman/listinfo/open-bio-announce) and [Twitter](https://twitter.com/obf_news).
 
-## What does the fellowship cover?
-The fellowship can be used to cover direct travel costs (such as airfare, train, metro, bus and taxi), hotel, child-care, remote video conferencing costs, and / or conference registration fees up to a value of USD 1,000. Note that rental car expenses and personal car mileage are not eligible for reimbursement. We anticipate that these awards will supplement travel costs, not cover all expenses. If you anticipate that you would require more than USD 1,000 in order to attend the event, please include that information on the application form.
+## Application form
 
-The OBF will reimburse the recipient after the event based on receipts provided and proof of attendance. There is no cash value beyond eligible and documented expenses. All reimbursement claims must be filed (and all prerequisites for receiving payment must be met, see below) within 3 months after conclusion of the event, after which the fellowship award expires.
+The applications are submitted for each round via [this application form](https://forms.gle/evzK6wBN8XtnGLUMA).
```

Comment:
```suggestion
The applications are submitted for each round via [this application form](https://forms.gle/evzK6wBN8XtnGLUMA).

We have created [a word template](https://docs.google.com/document/d/1tgzkHS84L8m3RwYoL7X86Axb6p_OiqNWaoJpiV4q8S0/edit?usp=sharing) of the form, which can be used for creating a draft of your application. 
Please make a copy or download this document as a '.docx' file.

If Google Forms are not accessible from your geographical location, you can submit your application by sharing the filled version of this document by email to the fellowship chair.
```

Reply:
Can someone check the wording for the template?

Reply:
looks good to me!